ijPlugin/src/main/kotlin/org/jetbrains/research/tasktracker/tracking/webcam/WebCamUtils.kt

```diff
@@ -13,14 +14,17 @@ import org.opencv.imgcodecs.Imgcodecs
 import org.opencv.videoio.VideoCapture
 import java.io.File
 
-@Suppress("TooGenericExceptionCaught", "SwallowedException", "MagicNumber")
+object WebCamUtils {
+    val logger: Logger = Logger.getInstance(WebCamUtils::class.java)
+}
+
 fun collectAllDevices(): List<WebCamInfo> {
     OpenCV.loadShared()
 
     getBaseTestSelfiePath().also { File(it).mkdirs() }
     val devices = mutableListOf<WebCamInfo>()
 
-    for (deviceNumber in (0..10)) {
+    for (deviceNumber in (MIN_CAMERA_QUANTITY..MAX_CAMERA_QUANTITY)) {
```

Comment:
Here we choose the device, in the makePhoto function, we set up the camera, we cannot use the same consts here. Also it would be better to catch an error here and stop trying different device numbers if a device with the current device number does not exist. We already did it with the olf library but did not keep this approach when rewrites to this opencv version